### PR TITLE
start: add sharing to studio

### LINKED ISSUE #4498: Add exp sharing to Studio in DVC exp get started
> or we can wait until we have capability to create a PR from Studio (in that case, let's create a separate issue for it).

Maybe we could wait for that and the ability to remove exps from the UI, so the update would be to add Studio tabs to https://dvc.org/doc/start/experiments/experiment-collaboration ?

_Originally posted by @daavoo in https://github.com/iterative/dvc.org/issues/4440#issuecomment-1521364779_

### PR DESCRIPTION
**Pull request by dberenbaum**
Closes #4498

### CODE REVIEW

**Review by shcheklein — APPROVED** [review]
It's great, thanks @dberenbaum . One minor comment (not even that . It feels that it might be beneficial to mention on this page that pushing is not creating actual commits, wdyt?

File: content/docs/start/experiments/experiment-collaboration.md
Comment (daavoo) [inline content/docs/start/experiments/experiment-collaboration.md:57]: Not a blocker but feels like potentially misleading as the tabs are usually alternatives but in this case you would need to use one of the other 2 tabs to see what is shown here

Comment (dberenbaum) [reply content/docs/start/experiments/experiment-collaboration.md:57]: Would you suggest dropping it? It doesn't quite fit, but I think it's still useful to show what pushing looks like in Studio.

**Comment by dberenbaum** [comment]
@shcheklein Not sure I get exactly what you mean. Do you think people will think it will do something like create a branch on their GH? 

My initial reaction is that we don't talk at all about what push does here, so I'd worry that any clarification might add more confusion. Maybe we could explain this better in https://dvc.org/doc/user-guide/experiment-management/sharing-experiments#push-experiments?

**Comment by shcheklein** [comment]
> Do you think people will think it will do something like create a branch on their GH?

More like actual commits.

> My initial reaction is that we don't talk at all about what push does here, so I'd worry that any clarification might add more confusion. 

Yep, it might. If we do this it should be super gentle I guess ("It doesn't create commits, see this ...)"? 

Anyways, your call. Not a blocker. I was just trying to read it as if I didn't have any context and realized that no chances I could have thought about some special refs, etc :) And in that case push == commit / branch? Just thinking aloud here.

**Comment by dberenbaum** [comment]
Thanks @shcheklein! Let's come back to this question. Can't quite think of a good way to address it right now.

File: content/docs/start/experiments/experiment-collaboration.md
Comment (dberenbaum) [reply content/docs/start/experiments/experiment-collaboration.md:57]: Let's come back to it if needed. Going to merge to unblock this one.

**Pull request merged**
